cdk/lib/__snapshots__/cdk-stack.test.ts.snap

```diff
@@ -135,7 +135,7 @@ Object {
                 "MetricName": "Errors",
                 "Namespace": "AWS/Lambda",
               },
-              "Period": 300,
+              "Period": 60,
```

Comment:
This (and the one below) is a change to the frequency at which to report to CloudWatch metrics. I can't see an explicit change for this within `@guardian/cdk` so I presume its a new default from `aws-cdk`.

This change means we're collecting 5 times more metrics, which might have a small cost increase...